app-i18n/fcitx-bamboo/fcitx-bamboo-1.0.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg cmake
+
+MY_PN="fcitx5-bamboo"
+MY_P="${MY_PN}-${PV}"
+S="${WORKDIR}/${MY_PN}-${PV}"
+DESCRIPTION="Typing Vietnamese by Bamboo core engine for Fcitx5"
+HOMEPAGE="https://github.com/fcitx/fcitx5-bamboo"
+
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_P}.tar.xz -> ${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="5"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	app-i18n/fcitx:5
+	sys-devel/gettext
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=dev-lang/go-1.18
+	kde-frameworks/extra-cmake-modules
+"
+pkg_postinst() {
+	[[ -z ${REPLACING_VERSIONS} ]] && return
+	elog "After Fcitx5-bamboo is installed, it is recommend to use GUI app to config language for IME"
+	elog " Run 'emerge -av app-i18n/fcitx-configtool' to modify typing language available"
+	elog "See https://wiki.gentoo.org/wiki/Fcitx for more info"
```

Comment:
If you write the pkg_postinst out and added your own commands, then the default behaveior of eclasses in this phase should also be written out. In other words, you should add `xdg_pkg_postinst` in the `pkg_postinst` phase here now, because your own `pkg_postinst` phase overwrites the default one defined in `xdg.eclass` .

Reply:
And, `-z` means the following variable is not set, so the following messages will display when this package is updated (disappear when first installed).

Reply:
I'm giving up on the suggestion package. Hope the user of this package will be smarter than me T^T

Reply:
@bekcpear hmmm, I think when the `REPLACEING_VERSOINS` is true, then all the command after `return` will be ignored. So it seems that he is correct.

Reply:
@Universebenzene I run depclean, then emerge new ebuild. It doesn't print out any text. Unless I have a variable active when INSTALL_NEW_PACKAGE (example like that)

Reply:
> @bekcpear hmmm, I think when the `REPLACEING_VERSOINS` is true, then all the command after `return` will be ignored. So it seems that he is correct.

You can test it.

Reply:
> @Universebenzene I run depclean, then emerge new ebuild. It doesn't print out any text. Unless I have a variable active when INSTALL_NEW_PACKAGE (example like that)

If you want show some notification, just replace `-z` with `-n`

Reply:
> You can test it.

I was wrong... When I change the `-z ` into `-n`, the postinst phase works as expected... I'm really bad at shell grammar 😭